src/terminal/parser/ut_parser/OutputEngineTest.cpp

```diff
@@ -1284,6 +1285,9 @@ class StatefulDispatch final : public TermDispatch
         case DispatchTypes::ModeParams::ASB_AlternateScreenBuffer:
             fSuccess = fEnable ? UseAlternateScreenBuffer() : UseMainScreenBuffer();
             break;
+        case DispatchTypes::ModeParams::XTERM_BracketedPasteMode:
```

Comment:
I see no test case for like `Win32InputMode`. So I didn't add one for `BracketedPasteMode`. Adding one would be trivial, though.